### PR TITLE
Polyhedron_demo: Fixes 

### LINKED ISSUE #2222: seg fault segmentation plugin
## Issue Details

Load a mesh,  Operations -> Segmentation.  Hit the Segment button results in a segfault

## Environment

* Operating system Windows 32/64 bits

* CGAL version: master

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

This PR renames some actions, capitalizes some text and fixes some problems, among which :
-  aabb_tree_items are not created with every new item when there is no cutting plane anymore
- Selection reducing/expanding works with Selection_SM_items 
## Release Management

* Affected package(s):Polyhedron_demo
* fixes #2222

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Plugins/Convex_decomposition/Nef_plugin.cpp [hunk 1]
```diff
@@ -127,14 +127,14 @@ Polyhedron_demo_nef_plugin::on_actionToNef_triggered()
...
-    new_nef_item->setName(tr("%1 (to nef)").arg(item->name()));
+    new_nef_item->setName(tr("%1 (Nef_polyhedron_3)").arg(item->name()));
     new_nef_item->setRenderingMode(item->renderingMode());
     item->setVisible(false);
   }
   else
   {
     new_nef_item = Scene_nef_polyhedron_item::from_polygon_mesh(sm_item);
-    new_nef_item->setName(tr("%1 (to nef)").arg(sm_item->name()));
+    new_nef_item->setName(tr("%1 (Nesf_polyhedron_3)").arg(sm_item->name()));
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Plugins/Convex_decomposition/Nef_plugin.cpp]: typo

File: Polyhedron/demo/Polyhedron/Scene.cpp [hunk 2]
```diff
@@ -45,8 +45,8 @@ Scene::Scene(QObject* parent)
                                       double, double, double)),
             this, SLOT(setSelectionRay(double, double, double,
                                        double, double, double)));
-    connect(this, SIGNAL(itemAboutToBeDestroyed(CGAL::Three::Scene_item*)),
-            this, SLOT(s_itemAboutToBeDestroyed(CGAL::Three::Scene_item*)));
+    //connect(this, SIGNAL(itemAboutToBeDestroyed(CGAL::Three::Scene_item*)),
+    //        this, SLOT(s_itemAboutToBeDestroyed(CGAL::Three::Scene_item*)));
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Scene.cpp]: Do not comment code but remove it.

**Comment by maxGimeno** [comment]
There is a bug with the selected faces normals in fairing_plugin.

**Comment by afabri** [comment]
tested successfully with VC++.

**Pull request merged**